aten/src/ATen/native/Math.h

```diff
@@ -1476,15 +1476,11 @@ calc_i0(T _x) {
   T x = std::abs(_x);
 
   if (x <= T{8.0}) {
-    auto coeff_pair = chebyshev_coefficients_i0e_A<T>();
-    auto A = std::get<0>(coeff_pair);
-    auto len = std::get<1>(coeff_pair);
+    auto [A, len] = chebyshev_coefficients_i0e_A<T>();
     T y = (x / T{2.0}) - T{2.0};
     return static_cast<T>(std::exp(x) * chbevl(y, A, len));
   }
-  auto coeff_pair = chebyshev_coefficients_i0e_B<T>();
-  auto B = std::get<0>(coeff_pair);
-  auto len = std::get<1>(coeff_pair);
+  auto [B,len] = chebyshev_coefficients_i0e_B<T>();
```

Comment:
Nit: I know we don't enforce it, but good to be consistent about spacing in the structured assignment.

Reply:
Fixed